### ISSUE TITLE
Why swap decoder during training?

### ISSUE THREAD

**Issue by zhanglonghao1992** [shaoanlu/faceswap-GAN#142]
Hi @shaoanlu , thaks for your amazing code.
I am a little confused about the training step of swapping decoders.
At about 4/5 of the training process,  you do:
`model.decoder_A.load_weights("models/decoder_B.h5") 
 model.decoder_B.load_weights("models/decoder_A.h5") `

Since decoder_A is used to transform a warped face into a fake face A. So why change it to decoder_B?

**Comment by yangcf10** [comment]
@zhanglonghao1992 Hi, have you figured out the reason why swapping decoders? I ran a few experiments and did not find it useful. For example, the loss for GA is higher after swapping decoders and cannot get back.
